### PR TITLE
Remove unused files and code, add versioned static

### PR DESCRIPTION
**Pull request by carkod**
## Done

Cleanup after https://github.com/canonical-web-and-design/netplan.io/pull/227
## QA

Everything should work as usual, compare / with  https://netplan.io/

### CODE REVIEW

File: templates/troubleshooting.html [hunk 1]
```diff
@@ -16,28 +16,28 @@ <h1 class="u-no-margin--bottom">Troubleshooting</h1>
   <div class="row">
     <div class="col-8 col-medium-4">
   <h2 id="troubleshooting-networking-issues">Troubleshooting networking issues</h2>
```
Comment (mtruj013) [inline templates/troubleshooting.html:18]: Is the indentation supposed to look like this? Everything that's been edited seems to be indented to the same number of spaces, are you seeing the same?

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Nice! The only difference I noticed was that this bottom border color is missing:
Screenshot 2022-06-15 at 14 05 54
Other than that there's the indentation thing, but I'm not sure if its just a weird GH issue or not? The hashes do indeed show up correctly when I inspect the demo 🙂

File: templates/troubleshooting.html (hunk 1 above)
Comment (carkod) [reply templates/troubleshooting.html]: Done

**Comment by carkod** [comment]
> Nice! The only difference I noticed was that this bottom border color is missing: Screenshot 2022-06-15 at 14 05 54 Other than that there's the indentation thing, but I'm not sure if its just a weird GH issue or not? The hashes do indeed show up correctly when I inspect the demo slightly_smiling_face

Done, but I think the styling of Vanilla navigation has changed the line to white

**Review by mtruj013 — APPROVED** [review]
LGTM! I'm still seeing some weird indentation, but other than that, all good 👍🏻

**Comment by carkod** [comment]
> LGTM! I'm still seeing some weird indentation, but other than that, all good 👍🏻

If you are talking about the `troubleshoot.html` file, this is because the `<pre>` sections need to be unindented to avoid adding spacing.

**Pull request merged**